index.js

```diff
@@ -1,5 +1,11 @@
+import {petCareItemsArray} from '/data.js'
+import {achievementItemsArray} from '/data.js'
```

Comment:
I think that can be shortened:
```suggestion
import { petCareItemsArray, achievementItemsArray } from '/data.js'
```